WNPRC_EHR/src/org/labkey/wnprc_ehr/WNPRC_EHRModule.java

```diff
@@ -235,18 +233,6 @@ protected void doStartupAfterSpringConfig(ModuleContext moduleContext) {
 
         BCReportRunner.schedule();
 
-//        for (Container studyContainer : getWNPRCStudyContainers()) {
```

Comment:
looks like you had this commented out, but the properties were still getting registered.